language/compiler/ir-to-bytecode/syntax/src/syntax.rs

```diff
@@ -1971,27 +1985,33 @@ fn parse_import_decl(
 
 // pub Module : ModuleDefinition = {
 //     "module" <n: Name> "{"
+//         <friends: (FriendDecl)*>
 //         <imports: (ImportDecl)*>
 //         <structs: (StructDecl)*>
 //         <functions: (FunctionDecl)*>
 //     "}" =>? ModuleDefinition::new(n, imports, structs, functions),
 // }
 
-fn is_struct_decl(tokens: &mut Lexer<'_>) -> Result<bool, ParseError<Loc, anyhow::Error>> {
-    let mut t = tokens.peek();
-    if t == Tok::Native {
-        t = tokens.lookahead()?;
-    }
-    Ok(t == Tok::Struct || t == Tok::Resource)
-}
-
 fn parse_module(
     tokens: &mut Lexer<'_>,
 ) -> Result<ModuleDefinition, ParseError<Loc, anyhow::Error>> {
+    fn is_struct_decl(tokens: &mut Lexer<'_>) -> Result<bool, ParseError<Loc, anyhow::Error>> {
+        let mut t = tokens.peek();
+        if t == Tok::Native {
+            t = tokens.lookahead()?;
+        }
+        Ok(t == Tok::Struct || t == Tok::Resource)
+    }
+
```

Comment:
Nit could you revert this. It will make some of my rebasing a bit more tedious

Reply:
Ah, sure, will do

Reply:
Done